src/Chapter1.hs

```diff
@@ -620,7 +635,13 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Integral a => a -> a
+sumLast2 n = last + secondToLast
+  where
+    absolute = abs n
+    last = absolute `mod` 10
+    withoutLast = absolute `div` 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: the standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(withoutLast, withLast) = absolute `divMod` 10
```

Reply:
Oh, nice! Didn't know that one before